src/viam/components/movement_sensor/movement_sensor.py

```diff
@@ -100,6 +100,8 @@ async def get_accuracy(
 
     async def get_readings(self, *, extra: Optional[Dict[str, Any]] = None, timeout: Optional[float] = None, **kwargs) -> Mapping[str, Any]:
         """Obtain the measurements/data specific to this sensor.
+        If a sensor is not configured to have a measurement or piece of data, it will be missing from the return dictionary.
```

Comment:
```suggestion

        If a sensor is not configured to have a measurement or fails to read a piece of data, it will be missing from the return dictionary.
```